ada/language/ast.py

```diff
@@ -6714,7 +6717,10 @@ def parent_name_equation(typ=T.BaseTypeDecl.entity, root=T.Name):
                 lambda ce=T.CallExpr:
                 ce.as_entity.subscriptable_type_equation(typ),
 
-                lambda ed=T.ExplicitDeref: ed.as_entity.eq_for_type(typ),
+                lambda ed=T.ExplicitDeref:
```

Comment:
Not clear what this is doing in this commit. Isn't that a fix in itself ? also not clear because this change is not commented, neither here or in the commit msg.

EDIT: Ok I understand now. Please comment what you are doing here (eg. storing a ref to the subp spec for later use in `is_call`, if I understand correctly)